src/trace_processor/perfetto_sql/stdlib/linux/cpu/utilization/thread.sql

```diff
@@ -144,3 +144,83 @@ JOIN _cpu_freq_per_thread AS c
   USING (id)
 GROUP BY
   utid;
+
+-- Aggregated CPU statistics for each thread per CPU combination.
```

Comment:
We've started asking people to leave a comment saying what data source you need to enable in the trace to have this table populated. Please add that in here.